README.md

```diff
@@ -9,7 +9,8 @@ About
 Requirements
 ------------
 
-* Django 1.6
+* Python 2.7
+* Django 1.8
```

Comment:
I know that we have no plans to pursue supporting Django 1.7, but we should add it here since we are testing against it in `tox.ini` and in `.travis.yml`, and the tests are passing.

Either that, or we should remove the test envs from the travis and tox